src/bin/hamcore/strtable_en.stb

```diff
@@ -552,6 +552,8 @@ HUB_AO_MaxLoggedPacketsPerMinute			Maximum number of logging target packets per
 HUB_AO_DoNotSaveHeavySecurityLogs			Do not take heavy security log.
 HUB_AO_DropBroadcastsInPrivacyFilterMode	Drop broadcasting packets if the both source and destination session is PrivacyFilter mode.
 HUB_AO_DropArpInPrivacyFilterMode			Drop ARP packets if the both source and destination session is PrivacyFilter mode.
+HUB_AO_AllowSameUserInPrivacyFilterMode			Allow packets if both the source and destination session users are the same, even in PrivacyFilter mode.

```

Comment:
Could you set this translation for `strtable_cn.stb`, `strtable_pt_br.stb` and `strtable_ru.stb`, please?

Reply:
Okay~